pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -3647,11 +3647,15 @@ ASTTable Table()  :
   <CREATE> [ ( <GLOBAL> | <PRIVATE> ) <TEMPORARY> | <SHARDED> | <DUPLICATED> ]
   <TABLE> ObjectNameDeclaration()
   [ <SHARING> "=" ( <METADATA> | <DATA> | <EXTENDED> <DATA> | <NONE> ) ]
+  //object_table
+  ( <OF> Datatype() |
+  //relational_table
   [ "(" ( LOOKAHEAD(2) OutOfLineConstraint() | TableColumn() ) ("," ( LOOKAHEAD(2) OutOfLineConstraint() | TableColumn() ))* ")" ]
   [LOOKAHEAD(4) <ON> <COMMIT> (<DROP> | <PRESERVE>) <DEFINITION>]
-  [LOOKAHEAD(4) <ON> <COMMIT> (<DELETE> | <PRESERVE>) <ROWS>]
-  //### [physicalProperties()]
-  //### [tableProperties()]
+  [LOOKAHEAD(4) <ON> <COMMIT> (<DELETE> | <PRESERVE>) <ROWS>] )
+  //### [PhysicalProperties()]
+  //### [TableProperties()]
+  Skip2NextOccurrence(";")
```

Comment:
Be aware, that this `Skip2NextOccurrence` does exactly what it is called: all tokens are swallowed until the next semicolon and are not parsed into a AST. This means, that whatever follows, is not accessible for rules.
I guess, you added this to fix parse errors? I'm ok with merging this, just want to make sure, you understand the limitations.